### DISCUSSION TITLE
如何实现审计字段禁止更新

### DISCUSSION THREAD

**Discussion by woodcoal** [dotnetcore/FreeSql#1710] [Q&A]
比如说实体 A，存在字段 a b c d，其中 b  字段数据更新时需要审核后才能发布。
我的想法是 Aop.AuditValue 时检查 b 是否存在，存在则记录到日志，b 更新取消。
但是好像实现不了，因为 Aop.AuditValue 没有取消字段更新功能，只能赋值新值。

不知道是否有其他解决方案可以实现呢？

**Comment by 2881099** [comment]
建议在上层业务上处理

**Reply by woodcoal** [reply]
看来也只能这样处理了 😀
